src/Stoa.ts

```diff
@@ -366,6 +368,51 @@ class Stoa extends WebService
             });
     }
 
+    /**
+     * GET /transaction/fees/:size
+     *
+     * Called when a request is received through the `/transaction/fees/:size` handler
+     *
+     * Returns transaction fees by the transaction size.
+     */
+    private getTransactionFees (req: express.Request, res: express.Response)
+    {
+        // TODO This is still incomplete.
+        //  We will proceed with statistics to find appropriate fees and then upgrade.
+
+        let size: string = req.params.size.toString();
```

Comment:
How about inputting the size as a `number`?

Reply:
In the following row, check to see if it is a positive number. 
It is only possible when it is a string